optimum/neuron/pipelines/transformers/sentence_transformers.py

```diff
@@ -0,0 +1,99 @@
+from typing import Dict
+
+from transformers.pipelines.base import GenericTensor, Pipeline
+
+from optimum.utils import is_sentence_transformers_available
+
+
+config_name = "config_sentence_transformers.json"
+
+
+def is_sentence_transformer_model(model: str, token: str = None, cache_folder: str = None, revision: str = None):
+    """Checks if the model is a sentence transformer model based on provided model id"""
+    if is_sentence_transformers_available():
+        from sentence_transformers.util import load_file_path
+
+        try:
+            load_file_path(
```

Comment:
This method shall also be applied to other non sentence transformers models isn't it? I think if we keep with the task name "feature-extraction" then we shall let the users tell or infer ourselves the library name infer_library_from_model which class to use with `TasksManager.infer_library_from_model`: https://github.com/huggingface/optimum/blob/e3fd2776a318a3a7b9d33315cc42c04c181f6d2f/optimum/exporters/tasks.py#L1690

Reply:
Thank you for sharing the TaskManager. I wasn't aware of that. I ll update the method